### PR TITLE
🐛  [8783] Fix Broken HelmCharts v0.33.11

### PR DESCRIPTION
**Pull request by mohamagdy**
## What
Fixed broken HelmCharts

## How
*Describe the solution*

## Recommended reading order
1. `charts/airbyte/templates/_helpers.tpl`
2. `charts/airbyte/values.yaml` <- this duplicates the `logs.minio.*` but I didn't find a better way to do it

## 🚨 User Impact 🚨
Users having a broken HelmChart v0.33.11 will be happy with this fix.

## Pre-merge Checklist
Maybe a another test with an external MinIO configurations? I tested with a local MinIO `minio.enabled: True`

#### Community member or Airbyter
   
- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [x] Documentation updated 
    - [x] Connector's `README.md`
    - [x] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [x] `docs/SUMMARY.md`
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] `docs/integrations/README.md`
    - [x] `airbyte-integrations/builds.md`
- [x] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
   
</p>
</details>

### CODE REVIEW

**Comment by marcosmarxm** [comment]
@mohamagdy thanks for the contribution, can you solve the file conflict?

**Comment by mohamagdy** [comment]
@marcosmarxm thank you too for taking a look. I fixed the conflicts (I let the `port` to be an integer value instead of `string` as it is in `master`).

File: charts/airbyte/values.yaml [hunk 1]
```diff
@@ -549,3 +549,9 @@ logs:
   gcs:
     bucket: ""
     credentials: ""
+
+minio:
+  accessKey:
+    password: minio
+  secretKey:
+    password: minio123
```
Comment (marcosmarxm) [inline charts/airbyte/values.yaml:557]: where are you planning to use this? The minio endpoint use `externalMinio` at the moment.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

**Comment by marcosmarxm** [comment]
@jrhizor can you review this change? PR https://github.com/airbytehq/airbyte/pull/8776 correct the problem casting the value to string. What do you think about this one?

**Review by jrhizor — APPROVED** [review]

**Review by marcosmarxm — APPROVED** [review]

File: charts/airbyte/values.yaml (hunk 1 above)
Comment (vnourdin) [reply charts/airbyte/values.yaml:557]: If I do a recursive search in the chart folder, I never find `.Values.minio` but only `.Values.logs.minio` so I don't understand how this is used ?
Are those values shared with the minio chart too ? If so, we cannot mutualize ak/sk with S3 and thus must refacto this.

Comment (vnourdin) [reply charts/airbyte/values.yaml:557]: If Values are used by the minio chart too it's normal indeed. I'll try to propose a PR to fix this without repeating later.

Comment (mohamagdy) [reply charts/airbyte/values.yaml:557]: Sounds good to me 👍

**Pull request merged**